src/traces/funnelarea/defaults.js

```diff
@@ -0,0 +1,76 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Lib = require('../../lib');
+var attributes = require('./attributes');
+var handleDomainDefaults = require('../../plots/domain').defaults;
+var handleText = require('../bar/defaults').handleText;
+
+module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout) {
+    function coerce(attr, dflt) {
+        return Lib.coerce(traceIn, traceOut, attributes, attr, dflt);
+    }
+
+    var len;
+    var vals = coerce('values');
+    var hasVals = Lib.isArrayOrTypedArray(vals);
+    var labels = coerce('labels');
+    if(Array.isArray(labels)) {
+        len = labels.length;
+        if(hasVals) len = Math.min(len, vals.length);
+    } else if(hasVals) {
+        len = vals.length;
+
+        coerce('label0');
+        coerce('dlabel');
+    }
+
+    if(!len) {
+        traceOut.visible = false;
+        return;
+    }
+    traceOut._length = len;
+
+    var lineWidth = coerce('marker.line.width');
+    if(lineWidth) coerce('marker.line.color', layout.paper_bgcolor);
+
+    coerce('marker.colors');
+
+    coerce('scalegroup');
+
+    var textData = coerce('text');
+    var textInfo = coerce('textinfo', Array.isArray(textData) ? 'text+value' : 'value');
```

Comment:
Here you're using `value` as the default, similar to `funnel` traces but unlike `pie` which uses `percent`.

I'd vote for making this align with `pie`: and using `percent` / `text+percent` as default values.

Reply:
cc: @nicolaskruchten

Reply:
Agreed with @etpinard

Reply:
Thanks @nicolaskruchten for the quick reply.

Reply:
Done in 8bd1875b390d91c07a8a8795b7eb9a84600c4ec9.